### PR TITLE
Swedish localization

### PR DESCRIPTION
**Pull request by NogginBops**

### CODE REVIEW

**Comment by Tranberry** [comment]
S**w**edish?

**Comment by NogginBops** [comment]
@Tranberry Woops...  :P

**Comment by vogonistic** [comment]
Some suggestions:

For `deconstruct_furniture` I think these are better: `Förstör Möbel` or `Ta isär Möbel`

`selection_panel_title=Nuvarande Markering`

Should `temp_pathfinding_test` maybe be `Ladda testvärld`? It's not just a pathfinding test anymore.

**Comment by NogginBops** [comment]
Aww, thought I squashed the commits...

File: Assets/StreamingAssets/Localization/sv_SE.lang [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿build_floor=Bygg Golv
+remove_floor=Ta bort Golv
+deconstruct_furniture=Dekonstruera Möbel
+new_world=Ny Värld
+load=Ladda
+save=Spara
+temp_pathfinding_test=Stigfinnings test
```
Comment (Tranberry) [inline Assets/StreamingAssets/Localization/sv_SE.lang]: Should it not be "ihopskrivet"? Stigfinnestest/Stigfinningstest

**Comment by NogginBops** [comment]
@vogonistic Good suggestions! Will add them.

**Comment by Sommerbo** [comment]
And this is why I prefer English over Swedish: Stigfinnings test

**Comment by Tranberry** [comment]
@vogonistic Why not "demontera möbel"?

**Comment by Mizipzor** [comment]
@NogginBops you can still squash commits! You just need to force push.

**Comment by Sommerbo** [comment]
"Skruva isär möbel" IKEA in SPAAAAACE

**Comment by NogginBops** [comment]
@Mizipzor Where do I do that?

File: Assets/StreamingAssets/Localization/sv_SE.lang (hunk 1 above)
Comment (NogginBops) [inline Assets/StreamingAssets/Localization/sv_SE.lang]: This is what we call "sär skrivning problem" in the business. :P

**Comment by Mizipzor** [comment]
@NogginBops you would do something like `git rebase -i HEAD~3` (or however many commits you want to squash) and then in the file that opens change some _pick_ to _squash_. Detail [here](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History). You should obviously be careful when doing this. But nothing is lost when its already online. After youve fixed it all, push with `--force`.

**Comment by NogginBops** [comment]
@Mizipzor I'm having a few problems with git atm. There are some file changes that won't go away even if I discard them. Because of that it's not letting me do much.

**Comment by vogonistic** [comment]
@Tranberry Just felt like a weird thing to say. Don't think I've ever used the word "demontera", even when I set up demos for our company at tradeshows. It was always "ta ner" eller "packa ihop".

**Comment by Tranberry** [comment]
@vogonistic might be region specific then.

**Comment by vogonistic** [comment]
@Tranberry Or "tryck ner skiten i lådan så vi kan gå hem".

I'm from Norrland, but I worked with people from all over the country. It just didn't feel very natural to say to me.

**Comment by NogginBops** [comment]
@vogonistic @Tranberry What about "plocka isär"?

**Comment by Sommerbo** [comment]
"Plocka isär" works you could drop "möbel" and let context work out, further I am happy with "möbel" as a generic term for placeable object.

**Comment by quill18** [comment]
Merged!

**Pull request merged**